Makefile

```diff
@@ -72,10 +72,28 @@ TAG ?= latest
 CONTROLLER_GEN="${GOBIN}/controller-gen"
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 
+# The CONTAINER_ENGINE variable is used for specifying the container engine. By default 'docker' is used
+# # but this can be overridden when calling make, e.g.
+# # CONTAINER_ENGINE=podman make container-image
+CONTAINER_ENGINE ?= docker
+IMAGE_REGISTRY ?= ghcr.io/shipwright-io
+SHIPWRIGHT_VERSION = latest
+DEVELOPER_TOOLS_DOCKERFILE = images/developer-tools/Dockerfile
+IMAGE_VERSION = $(shell scripts/hash-files.sh $(DEVELOPER_TOOLS_DOCKERFILE) Makefile | cut -c 1-12)
+CONTAINER_IMAGE = $(IMAGE_REGISTRY)/shipwright-builder-image:$(SHIPWRIGHT_VERSION)-$(IMAGE_VERSION)
+CONTAINER_RUN = $(CONTAINER_ENGINE) run --rm --interactive --tty --volume $(CURDIR):/src
```

Comment:
Imo overcomplicated wrt the amount of variables used here.

```suggestion
# The CONTAINER_ENGINE variable is used for specifying the container engine. By default 'docker' is used
# # but this can be overridden when calling make, e.g.
# # CONTAINER_ENGINE=podman make container-make
CONTAINER_ENGINE ?= docker
BUILDER_IMAGE ?= ghcr.io/shipwright-io/shipwright-builder-image:latest
```